docs/can-guides/introduction/technical.md

```diff
@@ -180,7 +180,7 @@ let fullName = value.returnedBy(function(){
 or like:
 
 ```js
-Person = DefineMap.extend({
+let Person = DefineMap.extend({
```

Comment:
`let` makes me think this is going to be reassigned somewhere. In the future, I think we should try to consistently use `const` unless something’s going to be reassigned.

Reply:
Sure, I was going to use `const` but changed my mind after seeing more uses of `let` :( (not a good reason)